doc/cephfs/fs-volumes.rst

```diff
@@ -134,6 +113,33 @@ The output format is json and contains fields as follows.
 * used_size: Current used size of the CephFS volume in bytes
 * pending_subvolume_deletions: Number of subvolumes pending deletion
 
+Sample output of volume info command::
+
+  $ ceph fs volume info vol_name
+  {
+      "mon_addrs": [
+          "192.168.1.7:40977"
+      ],
+      "pending_subvolume_deletions": 0,
+      "pools": {
+          "data": [
+              {
+                  "avail": 106288709632,
+                  "name": "cephfs.vol_name.data",
+                  "used": 4096
+              }
+          ],
+          "metadata": [
+              {
+                  "avail": 106288709632,
+                  "name": "cephfs.vol_name.meta",
+                  "used": 155648
+              }
+          ]
+      },
+      "used_size": 0
+  }
+
```

Comment:
I feel the existing layout would look better i.e. first the example then https://github.com/ceph/ceph/pull/48196/commits/183c3b9c2b6dad858e8a10fe0e81b88116e0f300#diff-45f83f814dee15dc10b8025029c5400a50e6c8972d5b6a795b62d62e0b278986R106-R114

Reply:
This is being done to maintain the uniformity.I have structured it the way, it is done for other commands where sample output has been provided.

Reply:
Yeah I just saw, snapshot cloning and subvolumes have it that way. Looks good.